batchflow/opensets/pascal.py

```diff
@@ -121,7 +136,23 @@ def download(self, path):
 
 
 class PascalClassification(BasePascal):
```

Comment:
Why did it lose VOC from its name?
```suggestion
class PascalVOCClassification(BasePascalVOC):
```